bentoml/cli/bento_service.py

```diff
@@ -96,20 +88,33 @@ def run_with_conda_env(bundle_path, command):
     return
 
 
-def make_bento_name_docker_compatible(name, tag):
+def make_bento_name_docker_compatible(name, version):
```

Comment:
let's separate this to two separate functions? `to_valid_docker_image_name` and `to_valid_docker_image_tag`?  Since the logic itself has nothing to do with BentoML